build/print.go

```diff
@@ -395,13 +395,14 @@ func (p *printer) expr(v Expr, outerPrec int) {
 		p.printf("%s", v.Name)
 
 	case *StringExpr:
-		// If the Token is a correct quoting of Value, use it.
-		// This preserves the specific escaping choices that
-		// BUILD authors have made, and it also works around
-		// b/7272572.
-		if strings.HasPrefix(v.Token, `"`) {
-			s, triple, err := unquote(v.Token)
-			if s == v.Value && triple == v.TripleQuote && err == nil {
+		// If the Token is a correct quoting of Value and has double quotes, use it.
+		// This preserves the specific escaping choices that BUILD authors have made,
+		// and it also works around b/7272572.
```

Comment:
replace b/ link with a github issue

Reply:
Maybe I should just remove it? This is an old bug that has been fixed already.

Reply:
Yes, remove.

Reply:
Done.